UI/org.eclipse.birt.report.designer.ui.editors/pom.xml

```diff
@@ -7,7 +7,6 @@
 		<version>4.8.0-SNAPSHOT</version>
 		<relativePath>../../</relativePath>
 	</parent>
-	<groupId>org.eclipse.birt</groupId>
```

Comment:
This change seems unnecessary, because it can inherit from parent.

Reply:
It's the other way around. -> Without this change Maven outputs a warning *because* the groupId can be inherited from the parent.